src/utils.jl

```diff
@@ -169,6 +169,16 @@ for (type, _, _) in ABSTRACT_QUANTITY_TYPES
 end
 Base.keys(q::UnionAbstractQuantity) = keys(ustrip(q))
 
+# If atol specified in kwargs, validate its dimensions and then strip units
+function _strip_kws(dimcheck, kws)
```

Comment:
Could we do a more descriptive name like `_validate_isapprox_kws`?

Reply:
Also should add an `@inline` to help force it

Reply:
Just commit an update with these comments addressed.